docs/reference/sql_adapter.rst

```diff
@@ -300,54 +300,167 @@ construct is mapped to PostgreSQL schema:
+          SET "global default::username" TO 'Tom'``;
+          SHOW "global default::username";
 
+    - Access policies are applied to object type tables when setting
+      ``apply_access_policies_pg`` is set to ``true``.
+
+    - Mutation rewrites and triggers are applied to all DML commands.
 
 DML commands
 ============
 
-When using ``INSERT``, ``DELETE`` or ``UPDATE`` on any table, mutation rewrites
-and triggers are applied. These commands do not have a straight-forward
-translation to EdgeQL DML commands, but instead use the following mapping:
+.. versionchanged:: _default
+
+    Data Modification Language commands (``INSERT``, ``UPDATE``, ``DELETE``, ..)
+    are not supported in EdgeDB <6.0.
+
+.. versionchanged:: 6.0
+
+.. versionadded:: 6.0
+
+    When using ``INSERT``, ``DELETE`` or ``UPDATE`` on any table, mutation
+    rewrites and triggers are applied. These commands do not have a
+    straight-forward translation to EdgeQL DML commands, but instead use the
+    following mapping:
 
-- ``INSERT INTO "Foo"`` object table maps to ``insert Foo``,
+    - ``INSERT INTO "Foo"`` object table maps to ``insert Foo``,
 
-- ``INSERT INTO "Foo.keywords"`` link/property table maps to an
-  ``update Foo { keywords += ... }``,
+    - ``INSERT INTO "Foo.keywords"`` link/property table maps to an
+      ``update Foo { keywords += ... }``,
 
-- ``DELETE FROM "Foo"`` object table maps to ``delete Foo``,
+    - ``DELETE FROM "Foo"`` object table maps to ``delete Foo``,
 
-- ``DELETE FROM "Foo.keywords"`` link property/table maps to
-  ``update Foo { keywords -= ... }``,
+    - ``DELETE FROM "Foo.keywords"`` link property/table maps to
+      ``update Foo { keywords -= ... }``,
 
-- ``UPDATE "Foo"`` object table maps to ``update Foo set { ... }``,
+    - ``UPDATE "Foo"`` object table maps to ``update Foo set { ... }``,
 
-- ``UPDATE "Foo.keywords"`` is not supported.
+    - ``UPDATE "Foo.keywords"`` is not supported.
 
 
 Connection settings
 ===================
 
-SQL adapter supports a limited subset of PostgreSQL connection settings.
-There are the following additionally connection settings:
+SQL adapter supports most of PostgreSQL connection settings
+(for example ``search_path``), in the same manner as plain PostgreSQL:
+
+.. code-block:: sql
+
+    SET search_path TO my_module;
+
+    SHOW search_path;
+
+    RESET search_path;
+
+.. versionadded:: 6.0
+
+    In addition, there are the following EdgeDB-specific settings:
+
+    - settings prefixed with ``"global "`` set the values of globals.
+      Because SQL syntax allows only string, integer and float constants in
+      ``SET`` command, globals of other types such as ``datetime`` cannot be set
+      this way. Special handling is in place to enable setting UUID globals via
+      hex-encoded strings. To set globals of other types via SQL, it is
+      recommended to change the global to use one of the simple types instead,
+      and use appropriate casts where the global is used.
```

Comment:
Can we maybe provide an example here? Does SQL syntax not support boolean? Maybe we can show an example of a bool global?

Reply:
Yea, good idea.

Reply:
> Does SQL syntax not support boolean?

No. Strangely it does not. Even though there are a ton of settings that expect boolean values. But they just look for `'true'` or `'yes'`. Bizarre.

Reply:
Well, syntactically it cannot be a boolean. But for postgres's settings, they have a bunch of string settings that they allow: https://www.postgresql.org/docs/current/config-setting.html

I'll implement these for globals too.

Reply:
Ref #8131

Reply:
> No. Strangely it does not. Even though there are a ton of settings that expect boolean values. But they just look for 'true' or 'yes'. Bizarre.

Oh my god, I hate it. 🙈 

Thanks for the update!